content/applications/marketing/surveys/overview/create.rst

```diff
@@ -1,88 +1,324 @@
-========================
-Get Started with Surveys
-========================
+=================
+Survey essentials
+=================
 
-Surveys can be used for a range of purposes that can go from collecting customer feedback, evaluate
-the success of an event, measure how pleased customers are with your products/services, gauge
-whether employees are happy and satisfied with their work environment, and even to find out what
-your market is thinking.
+Companies often use surveys to collect valuable information from their customers and/or employees,
+which in turn, they use to make more informed business decisions.
 
-Get started
-===========
+Surveys can be used for a number of different purposes. They can be used to collect customer
+feedback, evaluate the success of a recent event, measure the satisfaction of customers (or
+employees), learn what the market is *really* thinking, and so much more.
+
+Getting started
+===============
+
+To begin, click :guilabel:`Create` on the :guilabel:`Surveys` dashboard.
 
-When creating your survey, choose a *Title* and *Category*. The *Category* field is used to know in
-which context the survey is being conducted, as different applications might use it for different
-purposes such as recruitment, certification, or employee appraisal.
+Odoo redirects the page to a blank survey template form. 
 
-.. image:: create/new_survey.png
+First, choose a :guilabel:`title` for the survey. Next, add a cover image to the survey by clicking
+the :guilabel:`photo icon`. However, choosing a photo is not required.
+
+.. image:: create/title-photo-icon.png
    :align: center
-   :alt: Form view of a survey with a title and category being chosen in Odoo Surveys
+   :alt: Survey title field and upload photo icon.
 
-Tab: Questions
---------------
+Below the :guilabel:`title` is an option to assign the responsibility of the survey to a user in
+the database. Next, are various tabs in which the survey can be created, customized, and
+configured. These tabs are labeled as follows: :guilabel:`Questions`, :guilabel:`Options`,
+:guilabel:`Description`, and :guilabel:`End Message`.
```

Comment:
Great catch. And, since the image its referring to was just replaced, I left this screenshot alone, and kept the updated version of the "questions-description-options.png" image.